acme/acme/errors.py

```diff
@@ -1,6 +1,12 @@
 """ACME errors."""
 from acme.jose import errors as jose_errors
 
+MYPY = False
+if MYPY:
```

Comment:
I'm not sure whether or not you wanted a review on this yet, but you can work around coverage failures by appending `# pragma: no cover` to these lines.